src/main/java/edu/unc/lib/boxc/migration/cdm/DestinationsCommand.java

```diff
@@ -126,14 +127,37 @@ public int status() throws Exception {
         }
     }
 
-    private void validateOptions(GenerateDestinationMappingOptions options) {
+    @Command(name = "add",
+            description = "Add custom destination for individual CDM ID or list of IDs")
+    public int add(@Mixin DestinationMappingOptions options) throws Exception {
+        initialize();
+        var destinationMappingExists = Files.exists(project.getDestinationMappingsPath());
+        if (!destinationMappingExists) {
+          outputLogger.info("FAIL: Destination mapping at path " + project.getDestinationMappingsPath()
+                  + " does not exist");
```

Comment:
When using the logger, its preferable to use variable substitution rather than concatenation (for readability, and in some cases it is more efficient)
```suggestion
          outputLogger.info("FAIL: Destination mapping at path {} does not exist", project.getDestinationMappingsPath());
```